### DISCUSSION TITLE
Can I change the default hex for black, red, green, blue, yellow, purple, cyan and white?

### DISCUSSION THREAD

**Discussion by rakshit087** [starship/starship#3095] [Q&A] [answered]
I was wondering if there was a way to change the default hex code for the standard terminal colors using the starship.toml file. 
That would make theming much easier, since it will allow us to create color palettes.

**Comment by chipbuster** [comment, marked as answer]
There is not.

However, if you're using the standard color set in format strings (e.g. "red", "green" "blue" etc. instead of direct color hexes like `#00FF00`), the escape codes used by starship should be picked up by the terminal emulator and interpreted by its theme. So the terminal emulator will set the theme for starship.

If you want to set a colorscheme for starship which is different from your terminal emulator's, there is currently no way to do this in an easy manner.

**Reply by rakshit087** [reply]
Got it thanks 👍
